features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/db2.sql

```diff
@@ -93,10 +93,21 @@ CREATE TABLE IDN_OAUTH2_ACCESS_TOKEN (
             ACCESS_TOKEN_HASH VARCHAR (512),
             REFRESH_TOKEN_HASH VARCHAR (512),
             IDP_ID INTEGER NOT NULL,
+            TOKEN_BINDING_REF VARCHAR (32) DEFAULT 'NONE' NOT NULL,
             PRIMARY KEY (TOKEN_ID),
             FOREIGN KEY (CONSUMER_KEY_ID) REFERENCES IDN_OAUTH_CONSUMER_APPS(ID) ON DELETE CASCADE,
             CONSTRAINT CON_APP_KEY UNIQUE (CONSUMER_KEY_ID,AUTHZ_USER,TENANT_ID,USER_DOMAIN,USER_TYPE,TOKEN_SCOPE_HASH,
-                                 TOKEN_STATE,TOKEN_STATE_ID,IDP_ID)) INDEX IN TS32K
+                                 TOKEN_STATE,TOKEN_STATE_ID,IDP_ID,TOKEN_BINDING_REF)) INDEX IN TS32K
+/
+
+CREATE TABLE IDN_OAUTH2_ACCESS_TOKEN_BINDING (
+            TOKEN_ID VARCHAR (255) NOT NULL,
+            TOKEN_BINDING_TYPE VARCHAR (32) NOT NULL,
+            TOKEN_BINDING_REF VARCHAR (32) NOT NULL,
+            TOKEN_BINDING_VALUE VARCHAR (1024) NOT NULL,
+            TENANT_ID INTEGER DEFAULT -1,
```

Comment:
Do we need a default value for TENANT_ID?

Reply:
Other tables follow the same approach.